src/components/App.js

```diff
@@ -8,33 +8,36 @@ import { CATEGORIES, TASKS } from "../data";
 // console.log({ CATEGORIES, TASKS });
 
 function App() {
-  const [onCategory, setOnCategory] = useState("All")
-  const [onDetails, setOnDetails] = useState("")
-  const [onTasks, setOnTasks] = useState(TASKS)
-  
+  const [onCategory, setOnCategory] = useState("All");
+  const [onTasks, setOnTasks] = useState(TASKS);
```

Comment:
remove onDetails, as tests do not pass that into `<NewTaskForm />` and its lack is causing the submitted value to fail